bugbot/rules/uplift_beta.py

```diff
@@ -82,6 +88,16 @@ def bug_handler(bug, data):
 
         return bugs_without_regr
 
+    def is_needinfo_on_assignee(self, flags, assignee):
+        for flag in flags:
+            if (
+                flag["name"] == "needinfo"
+                and flag["status"] == "?"
+                and flag["requestee"] == assignee
+            ):
+                return True
+        return False
```

Comment:
nit: you could use `any()`:
```suggestion
        return any(
            flag["name"] == "needinfo"
            and flag["status"] == "?"
            and flag["requestee"] == assignee
            for flag in flags
        )
```

Reply:
Done here: 95106d1164132fa2bd317ebf2355e3c2de61e8b2